models/en-IN.json

```diff
@@ -61,14 +61,29 @@
                         {
                             "name": "messagechannel",
                             "type": "channelnames"
+                        },
+                        {
+                            "name": "emoji",
+                            "type": "emoji"
+                        },
+                        {
+                            "name": "emojiTwo",
+                            "type": "emoji"
+                        },
+                        {
+                            "name": "emojiThree",
+                            "type": "emoji"
                         }
                     ],
                     "samples": [
+                        "send {messagepost} emoji {emoji} {emojiTwo} {emojiThree} to {messagechannel}",
+                        "send {messagepost} emoji {emoji} {emojiTwo} to {messagechannel}",
+                        "send {messagepost} emoji {emoji}  to {messagechannel}",
                         "send {messagepost} to {messagechannel}"
                     ]
                 },
                 {
-                    "name": "GetLastMessageFromChannelIntent",
+                    "name": "GetLastMessageFromChannel",
```

Comment:
Please change `GetLastMessageFromChannel` to `GetLastMessageFromChannelIntent` in `models`.